.travis.yml

```diff
@@ -37,14 +38,22 @@ install:
     - pip install nbsphinx
     - pip install coveralls
 
-    # install keras contrib manually as currently not available on pypi
-    - pip install git+https://github.com/keras-team/keras-contrib.git@master
-
     # for test pypi add dependency installs manually
     # - pip install --index-url --no-deps https://test.pypi.org/simple sktime_dl==0.2.dev1
     - pip install .
     - python setup.py build_ext -i
 
+    # install keras contrib manually as currently not available on pypi
+    # pip install git+https://github.com/keras-team/keras-contrib.git@master
+    
+    # now need to install keras-contrib for tf.keras instead of standalone keras
+    # not needed for the tf_version 2.1 env, but does not hurt either. investigate 
+    # conditional installation 
+    - git clone https://www.github.com/keras-team/keras-contrib.git
+    - cd keras-contrib
+    - python convert_to_tf_keras.py
+    - USE_TF_KERAS=1 python setup.py install
+		
```

Comment:
This tab is what is preventing Travis from running. It needs to be deleted. 
I don't have write permission to do that.